packages/js/explat/src/test/assignment-test.js

```diff
@@ -85,6 +85,28 @@ describe( 'fetchExperimentAssignment', () => {
 		} );
 		await expect( assignment ).toEqual( data );
 	} );
+
+	it( 'adds woo_wcadmin_install_timestamp to request args', () => {
```

Comment:
It's worth noting that the woo_wcadmin_install_timestamp that these tests find are always going to be `undefined`, because the global that the code references hasn't been created. I could conceivably mock it and verify that the value pulled from it is the same as the value that I put in it, but I'm not convinced that would be valuable.